docs/images/LeRobot_Notebook.ipynb

```diff
@@ -0,0 +1,373 @@
+        "\n",
+        "\n",
+        "print(\"**Important**: Use a valid directory/jobs name. Avoid numbers or special characters other than '_'.\")\n",
+        "print(\"Example: 'training_results_aloha' or 'aloha_training_output'\")\n",
+        "\n",
+        "job_name = input(\"Please enter the job name for this training session: \")\n",
+        "\n",
+        "# Output directory with naming format instructions\n",
+        "\n",
+        "output_dir = input(\"Enter the output directory name: \")\n",
+        "\n",
+        "# Resume flag with disclaimer\n",
+        "resume_flag = input(\"Do you want to resume training from a previous checkpoint? (yes/no): \")\n",
+        "resume_cmd = \"--resume\" if resume_flag.lower() == 'yes' else \"\"\n",
+        "\n",
+        "# Model upload flag\n",
+        "upload_choice = input(\"Do you want to upload the model to Hugging Face after training? (yes/no): \")\n",
+        "model_repo_id = \"\"\n",
+        "if upload_choice.lower() == 'yes':\n",
+        "    model_repo_id = input(\"Please enter the model repo ID to store your trained model to Hugging Face (e.g., TrossenRoboticsCommunity/aloha_static_logo_assembly): \")\n",
+        "\n",
+        "# Local storage flag\n",
+        "store_locally = input(\"Do you want to store the training outputs locally? (yes/no): \")\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "6350b26b",
+      "metadata": {
+        "id": "6350b26b"
+      },
+      "source": [
+        "\n",
+        "## Step 1: GPU Setup & Compute Units\n",
+        "\n",
+        "The GPU type you selected earlier will now be configured for this cloud-based training session. Make sure to have enough compute units to support long training sessions, and monitor the first few epochs to ensure smooth execution.\n",
+        "\n",
+        "---\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "46a58617",
+      "metadata": {
+        "id": "46a58617"
+      },
+      "source": [
+        "\n",
+        "## Step 2: Installing Dependencies\n",
+        "\n",
+        "In this step, we'll install all the necessary dependencies for running LeRobot and performing model training.\n",
+        "\n",
+        "Ensure that these packages are successfully installed before proceeding to the next steps.\n",
+        "\n",
+        "---\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "49888a14",
+      "metadata": {
+        "id": "49888a14"
+      },
+      "outputs": [],
+      "source": [
+        "\n",
+        "# Install required dependencies\n",
+        "print(\"Installing required dependencies...\")\n",
+        "\n",
+        "!sudo apt-get install libusb-1.0-0-dev\n",
+        "!pip install --upgrade  pyrealsense2 dynamixel-sdk rerun-sdk blinker wandb datasets huggingface-hub hydra-core gitpython flask diffusers InquirerPy\n",
```

Comment:
Is there a requirements.txt file we can use instead?

Reply:
For some reason the requirements text file does not download the dependencies correctly. I'll come up with better solution in future.